validator/validator.go

```diff
@@ -78,6 +77,18 @@ func init() {
 		return hasValue(fl)
 	})
 
+	identifierRegex := regexp.MustCompile(identifierRegexString)
```

Comment:
change the name to `alphanumericunderscores`, we want to be as generic as possible